operators/hpe-csi-operator/deploy/operator.yaml

```diff
@@ -29,7 +29,7 @@ spec:
       containers:
         - name: hpe-csi-operator
           # Replace this with the built image name
-          image: quay.io/hpestorage/csi-driver-operator:v2.2.0
+          image: quay.io/hpestorage/csi-driver-operator:v2.3.0
```

Comment:
I can't find this tag on Quay, is that intentional?

Reply:
Yes, wanted add all the changes required for the 2.3.0 releae in this PR. will be pushing the operator image once the QA appove the final build.